app/src/androidTest/java/org/simple/clinic/storage/migrations/Migration89AndroidTest.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.storage.migrations
+
+import org.junit.Test
+import org.simple.clinic.assertViewExists
+
+class Migration89AndroidTest : BaseDatabaseMigrationTest(88, 89) {
+
+  @Test
+  fun the_overdue_patient_list_should_be_added() {
```

Comment:
Can we write another test verifying we receive the appointments with out phone number after the migration here?

Reply:
@vinaysshenoy It always feels a bit weird writing a test for Database view, we are essentially just writing a test to verify a view is present before and present after migration. While a test that verifies the behaviour after the migration is present in repository tests, should we also include those tests in migration? 🤔

Reply:
I think we don't write another test here as we already wrote a test for this data base view change in `PatientRepositoryAndroidTest` which works fine. And if our data base view changes are different from the migration the app will crash anyways. Therefore it might not be useful to test it again here.

Reply:
Yeah, but if we look at this migration test it doesn't actually help us determine what we are testing right? Basically in this test, we are testing the database view to exist before and after a migration. So it doesn't provide enough information on what exactly are we testing. At least that's what I feel like. But, yeah we don't have to hold back this PR for that. We can think about it later.